run_tests_with_chartests.sh

```diff
@@ -62,6 +62,9 @@ testrepo () {
     env GORACE='halt_on_error=1' go test -v -race -tags chartests $(cd $MODPATH && go list -m)
   done
 
+  # check linters
+  ./lint.sh
```

Comment:
good catch.  I forgot this script.  Although you'll want to remove the call to `golangci-lint` a few lines down.